CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,131 @@
+# Contributor Covenant Code of Conduct
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[no-file-name-irregular-characters] Do not use `_` in a file name](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-ios/pullRequest?prid=5645343)